clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -3647,7 +3679,7 @@ void handleResponse(AbstractResponse abstractResponse) {
 
                 // If coordinator changed since we fetched it, retry
                 if (ConsumerGroupOperationContext.hasCoordinatorMoved(response)) {
-                    Call call = getRemoveMembersFromGroupCall(context);
+                    Call call = getRemoveMembersFromGroupCall(context, allMembers);
```

Comment:
Why do we blindly put  `allMembers`? I believe we base on context to interpret, but like discussed earlier, this is easy to make mistake, we should rely on one source for members.

Reply:
Fixed, now we explicitly pass in the members to be deleted to the private `getRemoveMembersFromGroupCall`